erpnext/templates/pages/integrations/authorizenet_checkout.py

```diff
@@ -12,3 +12,4 @@ def get_context(context):
 	context.payment_context = json.dumps(payment_context)
 	context["description"] = payment_context.get("description") or ""
 	context["amount"] = fmt_money(amount=payment_context.get('amount'), currency=payment_context.get('currency'))
+	context["status"] = frappe.db.get_value("Payment Request", payment_context.get("reference_docname"), "status")
```

Comment:
Future proof this and use payment_context.get("reference_doctype") instead of hard coding "Payment Requests"